start-site/src/main/resources/application.yml

```diff
@@ -1292,4 +1292,4 @@ spring:
     restart:
       additional_paths:
         - start-client/build/
-      additional_exclude: "**/*.js,**/*.css"
\ No newline at end of file
+      additional_exclude: "**/*.js,**/*.css"
```

Comment:
Can you please remove that unrelated change?

Reply:
Remove the new line at the end.

Reply:
Not sure where this comes, maybe it comes from the old fork. If it is still an issue, I will redo the fork and make a new PR.